src/vm/docs/mod.rs

```diff
@@ -1449,6 +1449,58 @@ one of the following error codes:
 "
 };
 
+const GET_TOKEN_SUPPLY: SpecialAPI = SpecialAPI {
+    input_type: "TokenName",
+    output_type: "uint",
+    signature: "(ft-get-supply token-name)",
+    description: "`ft-get-balance` returns `token-name` circulating supply.
+The token type must have been defined using `define-fungible-token`.",
+    example: "
+(define-fungible-token stackaroo)
+(ft-mint? stackaroo u100 'SZ2J6ZY48GV1EZ5V2V5RB9MP66SW86PYKKQ9H6DPR)
+(ft-get-supply stackaroo) ;; returns u100
```

Comment:
Can you capitalize `Returns`? That way, the `docs::test::test_examples` test will check that the output matches expectation (or change the docs test to be case-insensitive in this detection).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2164/commits/44f7ade78ffff88cf2f23dadc8558ce8b7e0a642